packages/dev/s2-docs/pages/react-aria/getting-started.mdx

```diff
@@ -0,0 +1,186 @@
+import {Layout} from '../../src/Layout';
+export default Layout;
+
+import {InstallCommand} from '../../src/InstallCommand';
+import {Command} from '../../src/Command';
+import {BundlerSwitcher, BundlerSwitcherItem} from '../../src/BundlerSwitcher';
+import {StepList, Step, Counter} from '../../src/Step';
+import {ShadcnCommand} from '../../src/ShadcnCommand';
+import {StarterKits} from '../../src/StarterKits';
+import docs from 'docs:react-aria-components';
+import '../../tailwind/tailwind.css';
+
+export const section = 'Getting started';
+export const tags = ['introduction', 'installation'];
+export const description = 'Install and build your first component';
+
+# Getting started
+
+<PageDescription>How to install React Aria and build your first component.</PageDescription>
+
+## Install
+
+Install React Aria with your preferred package manager.
+
+<InstallCommand pkg="react-aria-components" />
+
+## Quick start
+
+The documentation for each component includes vanilla CSS and [Tailwind](https://tailwindcss.com) examples. Copy and paste these into your project and make them your own. You can also download each example as a ZIP or open in CodeSandbox or StackBlitz.
+
+<ExampleSwitcher>
+  ```tsx render docs={docs.exports.Select} links={docs.links} props={[]} type="vanilla" files={["starters/docs/src/Select.tsx", "starters/docs/src/Select.css"]} showCoachMark
```

Comment:
I like the idea of the coachmark, but I don't really like that it looks like a focus ring and that the focus ring when inside it is hard to see, maybe the coachmark ring could be pushed out by 2-4px and have the opacity lowered so it doesn't compete as much with the focus ring? or will that then fail contrast do you think?

not a high priority comment

Reply:
it's the Spectrum design for coachmark 🤷